src/pacasam/extractors/bd_ortho_vintage.py

```diff
@@ -44,64 +47,69 @@ class BDOrthoVintageExtractor(Extractor):
     """
 
     patch_suffix: str = ".tiff"
-    rgb_column: str = "rgb_file"
-    irc_column: str = "irc_file"
-    pixel_per_meter: int = 5
 
     def extract(self) -> None:
         """Download the orthoimages dataset."""
         iterable_of_args = []
-        for (rgb_file, irc_file), single_imagery in self.sampling.groupby([self.rgb_column, self.irc_column]):
+        for (rgb_file, irc_file), single_imagery in self.sampling.groupby([RGB_COLNAME, IRC_COLNAME]):
             iterable_of_args.append((rgb_file, irc_file, single_imagery))
 
         with WorkerPool(n_jobs=self.num_jobs) as pool:
             pool.map(self.extract_from_single_file, iterable_of_args, progress_bar=True)
 
     def extract_from_single_file(self, rgb_file, irc_file, single_file_sampling: GeoDataFrame):
-        with rasterio.open(rgb_file) as rgb, rasterio.open(irc_file) as irc:
+        with rasterio.open(rgb_file) as rgb_open, rasterio.open(irc_file) as irc_open:
             for patch_info in single_file_sampling.itertuples():
                 split = getattr(patch_info, SPLIT_COLNAME)
                 patch_id = getattr(patch_info, PATCH_ID_COLNAME)
                 tiff_patch_path: Path = self.make_new_patch_path(patch_id=patch_id, split=split)
                 if tiff_patch_path.exists():
                     continue
-
                 patch_geometry = getattr(patch_info, GEOMETRY_COLNAME)
```

Comment:
Is getattr() necessary here (also L63, L64)? "patch_info.geometry" doesn't work instead?

Reply:
You are right, will update.

Reply:
I prefer to keep constants to be coherent with all other colnames that are referenced via constants (like PATCH_ID_COLNAME, etc)